endpoint_search.go

```diff
@@ -22,6 +24,31 @@ const (
 	AvailabilityInternal Availability = "internal"
 )
 
+// ServiceTypeAliases contains a mapping of service types to any aliases, as
+// defined by the OpenStack Service Types Authority. Only service types that
```

Comment:
It would have been nice to provide a link (for each service) of the code where we can find the alias. But I'm fine as is now.